paper-radio-button.html

```diff
@@ -39,6 +39,9 @@
 `--paper-radio-button-unchecked-ink-color` | Selected/focus ripple color when the input is not checked | `--primary-text-color`
 `--paper-radio-button-checked-color` | Radio button color when the input is checked | `--default-primary-color`
 `--paper-radio-button-checked-ink-color` | Selected/focus ripple color when the input is checked | `--default-primary-color`
+`--paper-radio-button-size` | Size of the outer radio button | `16px`
+`--paper-radio-button-inner-size` | Size of the inner radio button | `50%`
+`--paper-radio-button-ripple-size` | Size of the radio button ripple | `48px`
```

Comment:
rather than have the ripple size be configured, what I did for checkbox (https://github.com/PolymerElements/paper-checkbox/blob/master/paper-checkbox.html#L84) was make it 2.66 times the button size:
```
calc(2.66 * var(--calculated-paper-checkbox-size));
```